packages/components/src/cookies_banner/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import Markdown from 'nebenan-components/lib/markdown';
+import Script from 'react-load-script';
+import styles from './index.module.scss';
+
+
+const CookiesBanner = ({
+  className,
+  children,
+  content,
+  GTMUrl,
+  insertGTM,
+  dismissed,
+  onScriptCreate,
+  ...rest
+}) => {
+  if (insertGTM) return <Script url={GTMUrl} onCreate={onScriptCreate} />;
+  if (dismissed) return null;
```

Comment:
this is probably not necessary/can be moved outside `condition && <Component />`

Reply:
it's still needs to render <Script />

Reply:
perhaps take `Scrip` out of here? would remove 4 props including `dismissed` and peer dependency